tasks/index.js

```diff
@@ -0,0 +1,226 @@
+// Task 1
+function checkNumbers(firstNumber, secondNumber) {
+    if(!isValidNumber(firstNumber) || !isValidNumber(secondNumber)) {
+        console.log("Please provide 2 numbers");
+        return;
+    }
+
+    let result = 'Both numbers are positive or negative';
+
+    if(firstNumber == 0 || secondNumber == 0) {
+        result = 'At least one of the numbers is unsigned(0)';
+    } else if (Math.sign(firstNumber) != Math.sign(secondNumber)) {
+        result = 'One of the numbers is positive and the other one is negative';
+    }
+
+    console.log(result);
+}
+
+// Task 2
+function getRandomNumber(min, max) {
+    return Math.floor(Math.random() * (max - min) + min);
+}
+
+function checkGuess() {
+    let numberToGuess = getRandomNumber(1, 11);
+    let userGuess = Number(prompt("The number is between 1 and 10(inclusive). Your guess: "));
+
+    if(!isValidNumber(userGuess)) {
+        alert("Please enter a number");
+        return;
+    }
+
+    let result = userGuess == numberToGuess ? 'Good Work' : 'Not matched';
+    
+    alert(result);
+}
+
+// Task 3
+let div = document.getElementById("page");
+let form;
+let inputFieldsArray = [];
+let calculateButton;
+let resultDiv;
+let h3;
+
+let page = {
+    createForm: function() {
+        if(!div.contains(form)) {
+            form = document.createElement("form");
+            div.appendChild(form);
+            console.log("Form created!");
+        }
+    },
+
+    createLabel: function() {
+        let label = document.createElement("label");
+        label.textContent = "Please enter a number: ";
+        return label;
+    },
+
+    createInputField: function(fieldsNumber) {
+        if(!isValidNumber(fieldsNumber)) {
+            console.log("Please enter a valid number as parameter");
+            return;
+        }
+
+        let fragment = document.createDocumentFragment();
+
+        for(let i = 0; i < fieldsNumber; i++) {
+            let inputField = document.createElement("input");
+            inputField.setAttribute("type", "number");
+            inputFieldsArray.push(inputField);
+            console.log("Input field created!");
+
+            if(!form.contains(calculateButton)) {
```

Comment:
Also add a text label with the number of the text label
Example: "number to multiply 4:"